common/switchboard.hpp

```diff
@@ -12,7 +12,11 @@
 #include <chrono>
 #include <exception>
 #include "phonebook.hpp"
-#include "cpu_timer.hpp"
+#if __has_include("cpu_timer.hpp")
+# include "cpu_timer.hpp"
```

Comment:
Extra space between `#` and `include`.

Reply:
This was intentional, because of [this](https://stackoverflow.com/questions/789073/indenting-defines), but actually I think I should just indent the whole line. I didn't know that was valid.